nuxeo/values.yaml

```diff
@@ -212,3 +212,5 @@ extraVolumes: []
 podSecurityContext:
   fsGroup: 1000
 containerSecurityContext: {}
+
+podDisruptionBudget: {}
```

Comment:
Any reason for not documenting the values? helm-docs is usually quite helpful for that purpose.

Reply:
Updated